MindscapeHQ/raygun4node
# chore: Release 0.15.0-0 🚀

## [chore]: Release `0.15.0-0` 🚀

### Description :memo:
- **Purpose**: We are releasing version `0.15.0-0` of the provider including resolving #146
- **Approach**: Merge all changes in `develop` into `master`

**Type of change**

**Release contents**
:point_right: Add support to async/await to `send` API method
:point_right: Fix avoiding infinite recursion in` filterKeys` method
:point_right: Various documentation and workflow changes
:point_right: Dev dependency updates 

### Test plan :test_tube:
- Run example apps and ensure that test exceptions are sending to Raygun
- Run the provider on clean project locally
- Use the `send` method in the example apps and clean project

### Author to check :eyeglasses:

### Reviewer to check :heavy_check_mark:
